.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: Node CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+        os: [ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
```

nit: I'd just change this to:
```suggestion
    runs-on: ubuntu-latest
```
and then remove line 11 above